Updated token response doc (#2559)
FusionAuth/fusionauth-site: site/docs/v1/tech/oauth/endpoints.adoc

This field will be returned in the response when `offline_access` scope was requested unless refresh tokens have been disabled for this application.
See `oauthConfiguration.generateRefreshTokens` in the link:/docs/v1/tech/apis/applications[Application API] or the `Generate refresh tokens` toggle in the FusionAuth UI when editing an application.

[field]#refresh_token_id# [type]#[UUID]#::

> The unique ID of the refresh token. This ID can be used to retrieve or revoke a specific refresh token when using the link:/docs/v1/tech/apis/jwt[Refresh Token API].

Reviewer: ```suggestion
The unique Id of the refresh token. This Id can be used to retrieve or revoke a specific refresh token using the link:/docs/v1/tech/apis/jwt[Refresh Token API].
```

+

> This field will be returned in the response when `offline_access` scope was requested unless refresh tokens have been disabled for this application.

Reviewer: ```suggestion
This field will be returned when the `offline_access` scope was requested and refresh tokens are allowed for this application.
```

> See `oauthConfiguration.generateRefreshTokens` in the link:/docs/v1/tech/apis/applications[Application API] or the `Generate refresh tokens` toggle in the FusionAuth UI when editing an application.

Reviewer: ```suggestion
Whether a refresh token is allowed for an application can be controlled by the   `oauthConfiguration.generateRefreshTokens` field in the link:/docs/v1/tech/apis/applications[Application API] or the `Generate refresh tokens` toggle in the FusionAuth admin UI.
```

Author: This was copied from the `refresh_token` description. Do you want this suggested copy used for both?

Reviewer: Yes please.

[field]#scope# [type]#[String]#::

> The space-separated list of scopes that were granted to this access token. This will only be present for grant types of `password` and `authorization_code`.

Reviewer: ```suggestion
The space-separated list of scopes allowed by the user during the grant which generated this access token. This will only be present for Authorization Code and Password grants.
```

I think that is a bit clearer.
